### PR TITLE
Fix tab navigation styles

### LINKED ISSUE #1694: Tabnav in the editing view is no longer styled correctly
**Describe the bug**

The Tab nav in the editing view is no longer styled correctly. At least, I'm pretty sure it was working properly before the upgrade to the new Primer version.

**To Reproduce**
1. Go to landing page
2. Click on 'Edit'
3. Observe the Tab navigation Edit / Preview

**Expected behavior**

Only the selected tab is shown.

**Screenshots**

* Current behavior

Screenshot 2021-03-02 at 22 15 47

* Expected behavior 

Screenshot 2021-03-02 at 22 16 04

**Environment Info**

```
Gollum 5.2.1
Running on: java with Ruby version 2.6.5
Using:
rjgit 5.7.0.2
gollum-rjgit_adapter 0.6
gollum-lib 5.1
Markdown rendering gem: kramdown
Other renderering gems:
none
```

### PR DESCRIPTION
**Pull request by benjaminwil**
Fixes #1694

The Primer CSS-provided `tabnav` styles were not being used on the Edit and Preview tabs on the create and edit pages.

After following [Primer's documentation][1], it looks like we were using the `aria-current` attribute incorrectly. Dynamically adding/removing this attribute on the selected tab fixes the issue.

[1]: https://primer-css-git-next-inputs.primer.now.sh/css/components/navigation#tabnav

### CODE REVIEW

File: lib/gollum/templates/create.mustache [hunk 1]
```diff
@@ -4,13 +4,15 @@
...
 <div id="wiki-content" class="create edit">
-	<div class="tabnav">
-	  <nav class="tabnav-tabs" aria-label="Foo bar">
-	    <a href="#" id="edit" class="tabnav-tab selected" aria-current="edit">Edit</a>
-	    <a href="#" id="preview" class="tabnav-tab" aria-current="preview">Preview</a>
-	  </nav>
-	</div>
-	
+  <div class="tabnav">
+    <nav class="tabnav-tabs" aria-label="Foo bar">
```
Comment (bartkamphorst) [inline lib/gollum/templates/create.mustache:8]: While you're making edits here, could you maybe give the `aria-label` of the `nav` element a slightly better name than `Foo bar`? 😇

Comment (benjaminwil) [reply lib/gollum/templates/create.mustache:8]: Haha, of course. Not sure how this got by me!

Comment (benjaminwil) [reply lib/gollum/templates/create.mustache]: Resolved!

**Comment by benjaminwil** [comment]
This pull request has been rebased against `master`, and the issue highlighted by @bartkamphorst has been resolved.

**Review by bartkamphorst — APPROVED** [review]
Looks good to me!

**Pull request merged**